test/index.js

```diff
@@ -464,6 +464,31 @@ describe('def', function() {
                    'as its first argument; received 0.5'));
   });
 
+  it('supports "enum" types', function() {
```

Comment:
Nit: I'd use "enumerated" here (unquoted). We needn't save characters in this context. :)